### ISSUE TITLE
“A date without time needn’t have a time zone”

### ISSUE THREAD

**Issue by ticky** [davedelong/calendar_fallacies#36]
The iCalendar specification (RFC5545) states that a DATE must not have a time zone, this means that all-day events in one time zone can show on the wrong day in another time zone.

I’ve been repeatedly grumpy about this [on Twitter](https://twitter.com/ticky/status/862369786516258816) as it means that as someone inhabiting a sufficiently different time zone (17 hours different) than my coworkers, “all-day” events they create always appear in my calendar on the wrong day, and vice versa!

**Comment by tewha** [comment]
This is true, but so is the inverse! I filed sort of a counter to this as #60: If you're not collaborating across time zones, adding a time zone do a date can cause **extra** problems. List Christmas Day as December 25th in the wrong time zone, and it may shift to December 24th or December 26th if you look at it in a different time zone.
